README.md

```diff
@@ -0,0 +1,24 @@
+<h3 align="center">vacay quiz</h3>
+<h4 align="center"><i>Learning Made Fun</i></h4>
+<p align="center"><a href="https://vacay-quiz.netlify.app">Play quizzes »</a></p>
+
+## Overview
+Vacay quiz is a web application for playing quizzes based on various Indian destinations. Vacay quiz is publicly hosted on Netlify at https://vacay-quiz.netlify.app/.
+
+## Demo
+![image](https://user-images.githubusercontent.com/56336326/154799966-7213f9cc-d502-4b3b-8387-a7e8de89af89.png)
```

Comment:
Appreciation - Its great that you have used images to showcase your project. Really helps to get a visual representation of your idea.

Reply:
Thank you for the appreciation.